packages/core/src/sass/app/_common/layout/layouts.scss

```diff
@@ -28,6 +28,35 @@
         overscroll-behavior: none;
         z-index: 2;
     }
+    /* temporary landscape blocker */
+    .landscape-blocker {
+        display: none;
+        position: fixed;
+        overflow: hidden;
+        width: 100%;
+        height: 100%;
+        background: var(--general-main-1);
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        color: var(--text-prominent);
+        font-size: 1.6rem;
+        z-index: 9999;
+        font-weight: bold;
+
+        &__icon {
+            width: 59px;
+            height: 64px;
+        }
+        &__message {
+            padding: 1.6rem 11rem;
+            line-height: 1.4;
+        }
+        @media all and (orientation: landscape) {
+            display: flex;
+            /* Style adjustments for landscape mode goes here */
```

Comment:
Question: is this comment necessary here?

Reply:
Nope, it isn't. I just kept it there. :D